.github/workflows/main.yml

```diff
@@ -156,6 +156,9 @@ jobs:
     steps:
       # Setup .npmrc file to publish to npm
       - uses: actions/setup-node@v4
+        with:
+          node-version: '20.x'
+          registry-url: 'https://registry.npmjs.org'
```

Comment:
So it is mentioned here but I wonder why didn't the error pop up before: https://docs.github.com/en/actions/publishing-packages/publishing-nodejs-packages#publishing-packages-to-the-npm-registry
> Please note that you need to set the registry-url to https://registry.npmjs.org/ in setup-node to properly configure your credentials.

Reply:
We used to have it on previous releases. For some reason, I got rid of it in our recent CI refactoring PRs.
It's only tested during the release/publishing process.